flatpak_builder_lint/staticfiles/exceptions.json

```diff
@@ -1190,6 +1190,10 @@
     "org.freedesktop.appstream-glib": {
         "flathub-json-skip-appstream-check": "It is a command line package"
     },
+    "se.emijoh.mpw": {
+        "flathub-json-skip-appstream-check": "It is a command line package",
```

Comment:
no. a CLI app should have a valid appstream file.

Reply:
Will fix the appstream file instead! 630719e